ynput/OpenPype
# Maya: Add repair action to hidden joints validator

## Changelog Description
Joints Hidden is missing repair action, this adds it back

## Additional info
Paragraphs of text giving context of additional technical information or code examples.

## Testing notes:
1. enable the Validate Rig Joints Hidden for your current bundle under "ayon+settings://maya/publish/ValidateRigJointsHidden/enabled"
2. Create some visible joints in Maya scene.
3. Create a rig from selection
4. run validation
5. repair action should be available from Report tab
